src/encoded/static/components/item-pages/DefaultItemView.js

```diff
@@ -234,13 +234,15 @@ export default class DefaultItemView extends React.PureComponent {
      * Returns list of elements to be rendered between Item header and the list of properties (or Tabs).
      * May be extended/customized.
      *
-     * @returns {JSX.Element[]} By default, `Publications.ProducedInPublicationBelowHeaderRow` and `StaticHeaderArea` component instances.
+     * @returns {JSX.Element[]} By default, `Publications.PublicationBelowHeaderRow` and `StaticHeaderArea` component instances.
      */
     itemMidSection(){
-        return [
-            <Publications.ProducedInPublicationBelowHeaderRow {...this.props} produced_in_pub={this.props.context.produced_in_pub} key="publication-info" />,
-            <StaticHeadersArea context={this.props.context} key="static-headers-area" />
-        ];
+        return (
```

Comment:
Starting in React v16 it is now possible to wrap components in a React.Fragment instead of needing to render a list of components/elements as descendant(s) of some Component `<div>`.

It is used here only to pretty-ify the code a little and make output of `itemMidSection` HTML-like instead of passing around an array of HTML-like code (JSX).

TODO: Change JSDoc to `returns {JSX.Element}` (?)